src/subcommand/server.rs

```diff
@@ -426,16 +485,33 @@ impl Server {
 
     let inscriptions = index.get_inscriptions_on_output(outpoint)?;
 
-    Ok(
+    Ok(if accept_json.0 {
+      axum::Json(serde_json::json!({
+        "value": output.value,
```

Comment:
for json there is no inscriptions info like webpage && ( Sat Range info etc)


how about 
```
+        "inscriptions": inscriptions.iter().map(|inscription| {
+          serde_json::json!({
+            "href": format!("/inscription/{}", inscription),
+          })
+        }).collect::<Vec<_>>(),
```
etc here ?


e.g.
https://ordinals.com/output/4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3:0
^ have inscriptions & other infos

```
{"_links":{"self":{"href":"/output/4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3:0"},"transaction":{"href":"/tx/4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3"}},"address":"bc1q3xfux78s6hr2g5h3rgpqwlmwqqfczt3r20k644","script_pubkey":"OP_0 OP_PUSHBYTES_20 8993c378f0d5c6a452f11a02077f6e0013812e23","transaction":"4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3","value":546}
```
not yet

Reply:
after this change  = 
```
{
  "_links": {
    "self": {
      "href": "/output/4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3:0"
    },
    "transaction": {
      "href": "/tx/4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3"
    }
  },
  "address": "bc1q3xfux78s6hr2g5h3rgpqwlmwqqfczt3r20k644",
  "inscriptions": [
    {
      "href": "/inscription/4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3i0"
    }
  ],
  "script_pubkey": "OP_0 OP_PUSHBYTES_20 8993c378f0d5c6a452f11a02077f6e0013812e23",
  "transaction": "4ee303f05452a63c2578e42df1b3c1b163a98746182b8f07a8d33706287659d3",
  "value": 546
}
```

Reply:
Will add this!

Reply:
https://github.com/casey/ord/pull/2036/commits/3f44a1afe35135cd52c60200917d2d85a8b54f9b